src/odemis/util/driver.py

```diff
@@ -255,6 +256,36 @@ def guessActuatorMoveDuration(actuator, axis, distance, accel=DEFAULT_ACCELERATI
         speed = actuator.speed.value.get(axis, DEFAULT_SPEED)
     return estimateMoveDuration(distance, speed, accel)
 
+def estimate_stage_movement_time(stage: model.Actuator, start_pos: Dict[str, float],
```

Comment:
@pieleric @tmoerkerken I have added a more generic stage movement estimation function here, it supports multi-axes and independent vs sequential axes